recipe/meta.yaml

```diff
@@ -18,20 +18,20 @@ build:
 
 requirements:
   host:
-    - python >=3.4
+    - python
```

Comment:
Why did you remove this?

Reply:
I wasn't thinking straight, I caught the error when I re-read it...